packages/uni_app/lib/view/bus_stop_selection/widgets/bus_stop_search.dart

```diff
@@ -38,6 +38,13 @@ class BusStopSearch extends SearchDelegate<String> {
     ];
   }
 
+  @override
+  ThemeData appBarTheme(BuildContext context) {
+    return Theme.of(context).copyWith(
+      scaffoldBackgroundColor: Theme.of(context).colorScheme.surface,
```

Comment:
Isn't it easier to do this on theme.dart instead?

Reply:
It is :)